src/UI/Status/Main.hs

```diff
@@ -108,15 +108,26 @@ renderStatusbar :: WithContext w => w -> AppState -> Widget Name
 renderStatusbar w s = withAttr statusbarAttr $ hBox
   [ str "Purebred: "
   , renderContext s w
+  , padLeftRight 1 (str "[")
   , renderNewMailIndicator s
   , renderMatches s
+  , renderToggled s
+  , padLeft (Pad 1) (str "]")
   , fillLine
   , txt (
       titleize (focusedViewName s) <> "-"
       <> titleize (focusedViewWidget s) <> " "
       )
   ]
 
+-- TODO
```

Comment:
TODO?

Reply:
Ah thanks for picking it up. I was trying to remember this morning to create a refactor issue, since all of this is currently a bit too rigid.